docs/source/reference/glossary.rst

```diff
@@ -181,3 +181,26 @@ tag
 TensorBay SDK has the ability to tag the specific :ref:`reference/glossary:commit` in a dataset's history
 as being important. Typically, people use this functionality to mark release points (v1.0, v2.0 and so on).
 
+
+branch
+======
+
+Similar to git, a branch is a lightweight pointer to one of the commits.
+
+Currently, TensorBay only supports a single branch, which is the default "main" branch.
+
+As :ref:`commits<reference/glossary:commit>` are submitted, the main branch will point to the lastest commit being made.
+Every time a commit is made, the main branch pointer moves forward automatically.
```

Comment:
Maybe replace these two lines into one is a better idea, such like:
Every time a :ref:`reference/glossary:commit` is submitted, the main branch pointer moves forward automatically to the latest commit.